### PR TITLE
Create #95-les-service-model

### PR DESCRIPTION
**Pull request by themue**
Open for feedback.

### CODE REVIEW

File: ideas/95-les-service-model.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+## Preamble
+
+    Idea: #95-les-service-model
+    Title: Introduce LES Service Model
+    Status: Draft
+    Created: 2018-03-20
+
+## Summary
+
+Introduce the LES Service Model where LES clients can subscribe to higher quality services. Different
```
[the remark this answers is not in the document]
Comment (themue) [reply ideas/95-les-service-model.md]: Good hint, will add it.

File: ideas/95-les-service-model.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- for-profit LES servers can still give away their extra capacity for free (with lower priority than paid service),
+- free service is a good indicator of high bandwidth capacity and therefore the capability to provide good service, and
+- paying clients will prefer servers which already gave them free service so free service can act as an advertisement.
+
+See https://github.com/zsfelfoldi/ethereum-docs/blob/master/les/service_model.md.
+
+Once the service model is introduced changes to Status app are needed:
+
+- Choice between free and paid LES service
+- In case of paid LES service definition of demands for auction
```
[the remark this answers is not in the document]
Comment (themue) [reply ideas/95-les-service-model.md]: Ah, haven't been aware how much these two a tied to each other.

File: ideas/95-les-service-model.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (b00ris) [reply ideas/95-les-service-model.md]: My 5 cents: not only full node could be shh/mailbox node.

File: ideas/95-les-service-model.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    Status: Draft
+    Created: 2018-03-20
+
+## Summary
+
+Introduce the LES Service Model where LES clients can subscribe to higher quality services. Different
+payment models will be available, negotiation between client and full node is done via auctions based
+on demands and capacity.
+
+This way full node service provide are able to monetize the services they provide. LES will make the
```
Comment (b00ris) [inline ideas/95-les-service-model.md]: We have full, fast, light types of sync. When you mention `full node`, do you mean `server node` with full/fast sync mode?

File: ideas/95-les-service-model.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+### Minimum Viable Product
+
+Goal Date: TBD (before end of Q2/2018)
+
+Description:
+
+- [ ] New Ultra Light Client Mode is added to LES
+- [ ] Ability for user to choose between free and standard payed priority service
+- [ ] Negotiation between client and full node
```
Comment (b00ris) [inline ideas/95-les-service-model.md]: It's not clear me. Could you explain, what do you mean there?

File: ideas/95-les-service-model.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+See https://github.com/zsfelfoldi/ethereum-docs/blob/master/les/service_model.md.
+
+Once the service model is introduced changes to Status app are needed:
+
+- Migration from Infura to ULC
+- Choice between free and paid LES service
+- In case of paid LES service definition of demands for auction
+
+For Status it also could become interesting to become a service provider offering payed full node services not only
+for own clients. This business case has to be evaluated.
```
Comment (arnetheduck) [inline ideas/95-les-service-model.md]: Are there any examples calculations for what a business could look like available?

File: ideas/95-les-service-model.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+Goal Date: TBD (before end of Q2/2018)
+
+Description:
+
+- [ ] New Ultra Light Client Mode is added to LES
+- [ ] Ability for user to choose between free and standard payed priority service
+- [ ] Negotiation about subscription between client and full node by performing an auction
+- [ ] Booking of micropayment fees for subscribed standard payed priority service
+
+Outline of MVP and next milestones is currently under discussion with Zsolt.
```
Comment (arnetheduck) [inline ideas/95-les-service-model.md]: Is there a trail somewhere for these discussions? What are potential candidates?

Comment (themue) [reply ideas/95-les-service-model.md]: Not yet, we're just starting and he yesterday hasn't been available. So I asked via chat and will start a log. That's one of the artifacts I would place in a swarm directory (see discussion in #swarm channel yesterday).

**Pull request merged**